docs/plugins/discovery-azure.asciidoc

```diff
@@ -1,36 +1,36 @@
-[[discovery-azure]]
+[[discovery-azure-classic]]
 === Azure Discovery Plugin
 
-The Azure Discovery plugin uses the Azure API for unicast discovery.
+The Azure Classic Discovery plugin uses the Azure Classic API for unicast discovery.
```

Comment:
Maybe add a hint on the restrictions and also a deprecation warning with a link to the alternative?